src/components/AssetsSearch/index.jsx

```diff
@@ -23,49 +20,28 @@ export default class AssetsSearch extends React.Component {
     }
   }
 
-  submit(event) {
+  submit = () => {
     this.props.updateSearch(this.state.value, this.props.courseDetails);
-    event.preventDefault(); // prevent the page from navigating to the form action URL
```

Comment:
Does the Paragon component handle this behavior for us now?

Reply:
Correct. Paragon doesn't expose the event via its onSubmit callback prop, either, so there's nothing available on which to call `preventDefault()`.